Packs/Cognni/Integrations/Cognni/Cognni.py

```diff
@@ -0,0 +1,690 @@
+# import demistomock as demisto
+from CommonServerPython import *
+
+import urllib3
+import dateparser
+import traceback
+from typing import Any, Dict, List, Optional, Tuple, cast, Iterable
+
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+MAX_EVENTS_TO_FETCH = 20
```

Comment:
Could even be 200 as that is our recommended limit (can be seen [here](https://xsoar.pan.dev/docs/integrations/fetching-incidents#fetch-limit))